chapters/classes.tex

```diff
@@ -644,20 +644,20 @@ \subsection{Short Class Definitions}\label{short-class-definitions}
 
 A class definition of the form
 \begin{lstlisting}[language=modelica]
-class IDENT1 = IDENT2 class-modification;
+class IDENT1 = type-specifier class-modification;
 \end{lstlisting}
 
-is identical, except that IDENT2 may be replaceable and for the lexical
+is identical, except that name may be replaceable and for the lexical
```

Comment:
"name" is a bit ambiguous here I think, maybe use e.g. "the base class" instead?

Reply:
I changed to type-specifier, which is unambiguous, and added (the base-class) in parenthesis.